### DISCUSSION TITLE
Multiple Databases

### DISCUSSION THREAD

**Discussion by lavericklavericklaverick** [PromtEngineer/localGPT#705] [Ideas]
Hi, it would be really great if it was possible to set the name of the database during both ingest and runlocalgpt as an arg; something like --dbname=XXXX.

ps, amazing project! thanks!

**Comment by anabellechan** [comment]
Yes, you can do that using langchain's db connection format for pgvector-PostgresSql, or Qdrant vector database, and use flags to tag each database --postgres --qdrant
